### ISSUE TITLE
RNA or SCT counts for DESeq2 on pseudobulk data?

### ISSUE THREAD

**Issue by e-manduchi** [satijalab/seurat#6922]
Hello,
I have an integrated Seurat object encompassing multiple individuals from different conditions. The objects has two count assays: the RNA and the SCT assay (the latter obtained with PrepSCTFindMarkers).
I'm interested in assessing DE with DESeq2 on pseudobulk data. Would you recommend using the RNA or the SCT counts to generate pseudobulk data as DESeq2 input?
Thanks

**Comment by saketkc** [comment]
I would expect the corrected counts and raw counts to give similar results. That said, DESeq2 performs its own size factor calculation so you would expect _some_ differences though I haven't explicitly benchmarked (but pseudobulking does help generally irrespective).

**Comment by e-manduchi** [comment]
Thanks for your feedback. For the data I have been working with, the SCT counts tend to be much lower than the RNA counts. Often non zero RNA counts are transformed into 0 SCT counts and this impacts the DESeq2 analysis on the derived pseudobulk counts. For example, even though the correlation between normalized pseudobulk counts derived from RNA and derived from SCT is high in my data (0.97), the correlation between the resulting -log10(p) from DESeq2 goes down to 0.76, and that between the log2FC to 0.7. The resulting lists of DE genes, when one selects typical thredholds (e.g. 0.05 or adjusted p-values and 1 for absolute log2FC) are quite different.

**Comment by saketkc** [comment]
What is your `median(object$nCount_RNA)`? The default scale factor for `NormalizeData()` is 10,000 which means the count you use in the `data` slot are after they have been scaled to 10,000 (and normalized). The default analog of this scaling factor in sctransform is `median(object$nCount_RNA)`.

**Comment by e-manduchi** [comment]
Hello,
to clarify my workflow:
for each individual I created a seurat object and I normalized the data solely with SCTransform (not with NormalizeData). I then integrated the individual objects and used PrepSCTFindMarkers.
The resulting integrated object has:
median(seurat$nCount_RNA) = 9896
median(seurat$nCount_SCT)=5400
The DESeq2 analyses I mentioned were done using pseudobulk counts from a subset of the original object (using a subset of the individuals and a specific cell type). For the subset object I have:
median(subset$nCount_RNA)=17829
median(subset$nCount_SCT)=5279
